Makefile.am

```diff
@@ -28,6 +28,7 @@ distdir: $(DISTFILES)
 		mv .extra_dist.tmp '$(srcdir)/.extra_dist'; fi
 	$(MAKE) $(AM_MAKEFLAGS) distdir-am EXTRA_FILES="$$(tr '\n' ' ' < $(srcdir)/.extra_dist) .extra_dist"
 	test ! -x $(srcdir)/src/client/add-metainfo-releases || $(srcdir)/src/client/add-metainfo-releases --version '$(VERSION)' '$(distdir)'/src/client/*.metainfo.xml
+	sed -i "/^Version:/ s/0/$(VERSION)/" "$(distdir)/tools/cockpit.spec"
```

Comment:
ugh.

Reply:
Hmm, not sure what to respond.. :man_shrugging: maybe? :grin: This was more or less the status quo until commit b34acdfcb740. If we want this file to be a real spec, we have to replace "0", not a "more magic" placeholder.

Reply:
I guess I don't like the asymmetry here vis-a-vis the other packaging files, now.

Could we turn our other packaging assets back into non-`.in` files and put version 0 in them and `sed` them up in the same way?  That would bring us full circle more or less back to the place we started...

Let's maybe have a chat...

Reply:
I added an extra commit which moves to the same approach for Debian and Arch, so it's again symmetric.

Reply:
:+1: